src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginConfiguration.java

```diff
@@ -65,6 +66,7 @@
 
     @Restricted(NoExternalUse.class)
     @RestrictedSince("2.0")
+    @DataBoundConstructor
```

Comment:
I doubt JCasC will alway take restricted constructors into account. CC @timja @casz

Reply:
hmm yeah probably worth adding a test to show that what you expect is exported:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#model-export-test

(also a different way of testing it in this PR here using a fixture instead of CNode):
https://github.com/jenkinsci/configuration-as-code-plugin/pull/860

Reply:
I just removed the restriction so that there should be no problem